model/schema/core.yaml

```diff
@@ -138,7 +138,7 @@ classes:
       - date_last_modified
 
   Reagent:
```

Comment:
Does it make sense to move the `Reagent` stuff to the new `reagent.yaml` so all of that stuff is in one place?